docs/assets/js/main.js

```diff
@@ -0,0 +1,14 @@
+let acc = document.getElementsByClassName("video_accordion");
+var i;
+
+for (i = 0; i < acc.length; i++) {
```

Comment:
since 'i' will only be used in the for block, then you can make it for (let i =0; i < acc.length; i++)